main.py

```diff
@@ -1,104 +1,89 @@
 from datetime import datetime
 from enums.cameraEnums import CameraMode, CameraStatus
 from enums.eventEnums import EventType
-from flask import Flask, redirect, url_for, render_template, request, jsonify, flash
+from models.models import Users, Event, Camera
+from flask import Flask, Blueprint, redirect, url_for, render_template, request, jsonify, flash
 from flask_bootstrap import Bootstrap
-from flask_login import LoginManager, UserMixin, login_user, login_required, logout_user, current_user
-from flask_sqlalchemy import SQLAlchemy
+from flask_login import LoginManager, login_user, login_required, logout_user, current_user
 from Forms import LoginForm, SignUpForm
 from os import environ
 from werkzeug.utils import secure_filename
 from werkzeug.security import generate_password_hash, check_password_hash
 import boto3
 
+bp = Blueprint('myapp', __name__)
 
-app = Flask(__name__)
-app.config['SECRET_KEY'] = 'imageanalysissystem'
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:' + environ["DB_PASSWORD"] + '@lfiasdb.cwtorsyu3gx6.us-west-2.rds.amazonaws.com/postgres'
+loginManager = LoginManager()
 
+def create_app():
+    app = Flask(__name__)
 
-Bootstrap(app)
+    app.config['SECRET_KEY'] = 'imageanalysissystem'
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:' + environ["DB_PASSWORD"] + '@lfiasdb.cwtorsyu3gx6.us-west-2.rds.amazonaws.com/postgres'
 
-db = SQLAlchemy(app)
+    Bootstrap(app)
 
-loginManager = LoginManager()
-loginManager.init_app(app)
-loginManager.login_view = 'login'
-
-class Users(db.Model, UserMixin):
-    id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(50), nullable=False)
-    email = db.Column(db.String(50), unique=True, nullable=False)
-    password = db.Column(db.String(80), nullable=False)
-    events = db.relationship('Event', backref='users', lazy=True)
-    cameras = db.relationship('Camera', backref='users', lazy=True)
-
-class Event(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
-    camera_id = db.Column(db.Integer, db.ForeignKey('camera.id'), nullable=False)
-    type = db.Column(db.Enum(EventType), nullable=False)
-    timestamp = db.Column(db.DateTime, nullable=False)
-
-class Camera(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
-    name = db.Column(db.String(50), nullable=False)
-    status = db.Column(db.Enum(CameraStatus), nullable=False)
-    mode = db.Column(db.Enum(CameraMode), nullable=False)
-    
+    loginManager.init_app(app)
+    loginManager.login_view = 'myapp.login'
 
+    app.register_blueprint(bp)
```

Comment:
Along with the factory pattern, we have to use **blueprints** to add/edit routes.